services/auth/source/pam/source_authenticate.go

```diff
@@ -58,10 +59,12 @@ func (source *Source) Authenticate(user *user_model.User, userName, password str
 		LoginType:   auth.PAM,
 		LoginSource: source.authSource.ID,
 		LoginName:   userName, // This is what the user typed in
-		IsActive:    true,
+	}
+	overwriteDefault := &user_model.CreateUserOverwriteOptions{
+		IsActive: util.OptionalBoolTrue,
```

Comment:
OK to bypass setting.Service.RegisterEmailConfirm and setting.Service.RegisterManualConfirm here?